wpilibj/src/main/java/edu/wpi/first/wpilibj/Telemetry.java

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj;
+
+import edu.wpi.first.util.telemetry.TelemetryTable;
+
+/**
+ * Telemetry sends information from the robot program to dashboards, debug tools, or log files.
+ *
+ * <p>For more advanced use cases, use the NetworkTables or DataLog APIs directly.
+ */
+public final class Telemetry {
+  /** The root {@link TelemetryTable}. */
+  private static final TelemetryTable m_root = new TelemetryTable("/");
```

Comment:
Suggest protected instead of private so subclasses can get to it.

Reply:
Added a setter (with a warning about it being non-thread-safe) in case someone wants to customize this.

Reply:
As a utility class, it's not useful to subclass.  However, changing behavior is possible via the SendableRegistry backend.